yarn.lock

```diff
@@ -1426,6 +1426,11 @@
   resolved "https://registry.yarnpkg.com/@types/json-schema/-/json-schema-7.0.5.tgz#dcce4430e64b443ba8945f0290fb564ad5bac6dd"
```

Comment:
A bit surprising that it changed since I can't see anything in `package.json`, are can you run `yarn install` again to make sure it isn't caused by a package you (we) got rid of?